DMCompiler/DM/Expressions/Constant.cs

```diff
@@ -96,6 +112,28 @@ class Null : Constant {
             json = null;
             return true;
         }
+
+        public override Constant GreaterThan(Constant rhs) {
+            return new Number(Location, 0); // false
```

Comment:
`null > -1` is true

Reply:
Nice catch, didn't have negative numbers for the unit test. Nor a larger number.